x-pack/plugins/reporting/server/export_types/common/get_custom_logo.ts

```diff
@@ -20,21 +21,17 @@ export const getCustomLogo = async ({
   job: TaskPayloadPDF;
   conditionalHeaders: ConditionalHeaders;
 }) => {
-  const serverBasePath: string = config.kbnConfig.get('server', 'basePath');
-  const fakeRequest: any = {
-    headers: conditionalHeaders.headers,
-    // This is used by the spaces SavedObjectClientWrapper to determine the existing space.
```

Comment:
Previously (before NP), knowing the client-side basePath was important for also knowing the spaceId.